diff --git a/dist/cjs/index.development.js b/dist/cjs/index.development.js
index d66ede30dc83520441a67cc11e097c815e6119f1..60128fc8e6d2d22611f3be95ecacde5776f5b23d 100644
--- a/dist/cjs/index.development.js
+++ b/dist/cjs/index.development.js
@@ -1586,7 +1586,7 @@ const getUnifiedContentLine = (diffFile) => {
     const unifiedLines = [];
     numIterator(unifiedLineLength, (index) => {
         const unifiedLine = diffFile.getUnifiedLine(index);
-        if (!unifiedLine.isHidden) {
+        if (unifiedLine && !unifiedLine.isHidden) {
             unifiedLines.push({ type: exports.DiffFileLineType.content, index, lineNumber: index + 1, unifiedLine: unifiedLine });
         }
     });
diff --git a/dist/cjs/index.production.js b/dist/cjs/index.production.js
index 02eaaaf3bec6deb5fe34d003cc20b5ed5debc8d7..5082178c6c96720445c27be84965e162e88f6e78 100644
--- a/dist/cjs/index.production.js
+++ b/dist/cjs/index.production.js
@@ -1539,7 +1539,7 @@ const getUnifiedContentLine = (diffFile) => {
     const unifiedLines = [];
     numIterator(unifiedLineLength, (index) => {
         const unifiedLine = diffFile.getUnifiedLine(index);
-        if (!unifiedLine.isHidden) {
+        if (unifiedLine && !unifiedLine.isHidden) {
             unifiedLines.push({ type: exports.DiffFileLineType.content, index, lineNumber: index + 1, unifiedLine: unifiedLine });
         }
     });
diff --git a/dist/esm/index.mjs b/dist/esm/index.mjs
index 3c556870f2defb9f458f4a5d5561f15e955d6234..aa8e0cbb5c6482e0c297a09ebdf22d43965cf392 100644
--- a/dist/esm/index.mjs
+++ b/dist/esm/index.mjs
@@ -1585,7 +1585,7 @@ const getUnifiedContentLine = (diffFile) => {
     const unifiedLines = [];
     numIterator(unifiedLineLength, (index) => {
         const unifiedLine = diffFile.getUnifiedLine(index);
-        if (!unifiedLine.isHidden) {
+        if (unifiedLine && !unifiedLine.isHidden) {
             unifiedLines.push({ type: DiffFileLineType.content, index, lineNumber: index + 1, unifiedLine: unifiedLine });
         }
     });
